content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -1,195 +1,109 @@
-| kubeadm alpha subcommands | alpha          |
-| CoreDNS                   | alpha          |
-| DynamicKubeletConfig      | alpha          |
-
-
-kubeadm's overall feature state is **Beta** and will soon be graduated to
-**General Availability (GA)** during 2018. Some sub-features, like self-hosting
-or the configuration file API are still under active development. The
-implementation of creating the cluster may change slightly as the tool evolves,
-but the overall implementation should be pretty stable. Any commands under
-`kubeadm alpha` are by definition, supported on an alpha level.
-
-
-### Support timeframes
-
-Kubernetes releases are generally supported for nine months, and during that
-period a patch release may be issued from the release branch if a severe bug or
-security issue is found. Here are the latest Kubernetes releases and the support
-timeframe; which also applies to `kubeadm`.
-
-| Kubernetes version | Release month  | End-of-life-month |
-|--------------------|----------------|-------------------|
-| v1.6.x             | March 2017     | December 2017     |
-| v1.7.x             | June 2017      | March 2018        |
-| v1.8.x             | September 2017 | June 2018         |
-| v1.9.x             | December 2017  | September 2018    |
-| v1.10.x            | March 2018     | December 2018     |
+kubeadm's overall feature state is **Beta**. 
 
 {{% /capture %}}
 
 {{% capture prerequisites %}}
 
-1. One or more machines running a deb/rpm-compatible OS, e.g. Ubuntu or CentOS
-1. 2 GB or more of RAM per machine (any less will leave little room for your
-   apps)
-1. 2 CPUs or more on the master
-1. Full network connectivity between all machines in the cluster (public or
-   private network is fine)
-   
-{{< note >}}
-**Note:** This guide results in a Kubernetes cluster with one master and a
-number of nodes that you decide. A single master is not highly available. If
-you want to set up a multi-master cluster for high availability, you can
-follow
-[this guide instead](https://kubernetes.io/docs/setup/independent/high-availability/).
-{{< /note >}}
+- One or more machines running a deb/rpm-compatible OS, for example Ubuntu or CentOS
+- 2 GB or more of RAM per machine. Any less leaves little room for your
+   apps.
+- 2 CPUs or more on the master
+- Full network connectivity among all machines in the cluster. A public or
+   private network is fine.
  
 {{% /capture %}}
 
 {{% capture steps %}}
 
 ## Objectives
 
-* Install a secure Kubernetes cluster on your machines
+* Install a single master Kubernetes cluster or [high availability cluster](https://kubernetes.io/docs/setup/independent/high-availability/)
 * Install a Pod network on the cluster so that your Pods can
   talk to each other
 
 ## Instructions
 
 ### Installing kubeadm on your hosts
 
-See [Installing kubeadm](/docs/setup/independent/install-kubeadm/).
+See ["Installing kubeadm"](/docs/setup/independent/install-kubeadm/).
```

Comment:
What are these quotes about?

Reply:
A convention I’d like us to move towards when linking to other pages within the site. It’s out of scope for this PR, but we can have a larger conversation about it. 

>  For more information, see “This page”.